### DISCUSSION TITLE
Pijul support

### DISCUSSION THREAD

**Discussion by dantefromhell** [buildbot/buildbot#7676] [Ideas]
My impression is that Buildbot supports a wide range of open source source control systems which I really enjoy seeing.

I was wondering if pijul (https://www.pijul.com) would be a a candidate to support?

From what I understand it's inspired by darcs (which is already supported) but trying to have a changes work "correcter" (as in mathematical correctness) and avoid some of darcs performance issues.

**Comment by p12tic** [comment]
> I was wondering if pijul (https://www.pijul.com/) would be a a candidate to support?

All VCS are candidates to support. The limitation is whether there's someone willing to implement support for the feature and write extensive unit tests, so that maintainer can apply minor changes to the code without fearing breakage.
